lib/bigcommerce/resources/v3/email_template.rb

```diff
@@ -0,0 +1,13 @@
+module Bigcommerce
+  module V3
+    class EmailTemplate < Resource
+      include Bigcommerce::ResourceActions.new uri: 'marketing/email-templates/%s'
+
+      property :type_id
+      property :subject
+      property :body
+      property :translations
+      property :meta
```

Comment:
I think `:meta` property won't work, it is meta information about the response but not a part of `EmailTemplate` object.